replication_report-extensions.Rmd

```diff
@@ -0,0 +1,320 @@
+---
+title: "Replication Report"
+author: "Yari & Kyonne" 
+date: "April 26, 2015"
+output: html_document
+---
+
+(1) Replicated table:
+
+```{r echo = FALSE}
+
+# Load packages
+suppressPackageStartupMessages(require(dplyr))
+suppressPackageStartupMessages(require(tidyr))
+suppressPackageStartupMessages(require(pwr)) 
+suppressPackageStartupMessages(require(broom))
+suppressPackageStartupMessages(require(lsr))
+suppressPackageStartupMessages(require(ggplot2))
+suppressPackageStartupMessages(require(MBESS))
+suppressPackageStartupMessages(require(psych))
+suppressPackageStartupMessages(require(car))
+suppressPackageStartupMessages(require(MASS))
+suppressPackageStartupMessages(require(stargazer))
```

Comment:
Might seem silly and very trivial but I think there is a (heated) debate about using require vs. library. According to this post, library is a better option for loading packages: http://www.r-bloggers.com/library-vs-require-in-r/